src/test/java/com/bae/service/test/unit/service/FreezerServiceTest.java

```diff
@@ -81,5 +97,30 @@ public void deleteFreezerTest() throws FreezerDoesntexistException {
 		verify(this.repo, times(1)).deleteById(this.id);

```

Comment:
since your delete freezer method returns a boolean, maybe you could add an assertEquals to see if the delete method actually deletes the freezer.